lib/Mail.php

```diff
@@ -488,6 +488,10 @@ public function send(MailerInterface $mailer = null)
 
         $this->subject($this->getSubjectRendered());
 
+        // Remove the document property because it is no longer needed and makes it difficult
+        // to serialize the Mail object when using the Symfony Messenger component
+        $this->setDocument(null);
```

Comment:
please adapt `setDocument()` method to nullify `$this->document` property. At the moment, it will throw exception `document is not an instance of Pimcore\Model\Document\Email` https://github.com/pimcore/pimcore/blob/e814e9cb0ae8300a2cf27d1b78db391168a0c0c9/lib/Mail.php#L769-L774

Reply:
Ah, I added this to my local test version but forgot to add it to the PR